tests/savepoint/translate/translate_d_sw.py

```diff
@@ -182,9 +178,7 @@ def compute_from_storage(self, inputs):
         # TODO add these to the serialized data or remove the test
         inputs["damp_w"] = column_namelist["damp_w"]
         inputs["ke_bg"] = column_namelist["ke_bg"]
-        inputs["dt"] = (
-            spec.namelist.dt_atmos / spec.namelist.k_split / spec.namelist.n_split
-        )
+        inputs["dt"] = spec.namelist.dt_atmos / spec.namelist.k_split / spec.namelist.n_split
```

Comment:
has this been run through the linter? this reformatting is likely unintentional?